packages/styles/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@salt-ds/styles",
+  "version": "1.0.0-alpha.0",
```

Comment:
We should probably make this a 0.1.0 release so we can reserve the right to make breaking changes if we need to